Kernel/Devices/GPU/Intel/Definitions.h

```diff
@@ -8,7 +8,7 @@
 
 #include <AK/Types.h>
 
-namespace Kernel::IntelGraphics {
+namespace Kernel::IntelGPU {
```

Comment:
Same for this one (and subsequent uses of the namespace).

Reply:
I'd actually leave this as `IntelGPU`, because future generations are not merely about graphics (2D) but it's an entire processor unit.